client/src/components/JobInformation/JobError.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+    <b-card border-variant="danger" :header="header">
+        <b-card-text>
+            <div @click="showInfo = true">
+                <a href="#">See full job details <font-awesome-icon icon="info-circle" /></a>
+            </div>
+            <div class="error-wrapper" v-if="job.stderr" @click="toggleExpanded">
+                Job Standard Error
+                <pre :class="errorClasses">{{ job.stderr }}</pre>
+                <b v-if="!expanded">(Click to expand job standard error)</b>
+            </div>
+            <!-- TODO: modal for reporting error. -->
+        </b-card-text>
+        <b-modal v-model="showInfo" scrollable ok-only>
```

Comment:
b-modal leaves space for a title, we should maybe optionally disable the title in the JobInformation component and set a title here instead. That would probably look better than this:
<img width="754" alt="Screenshot 2021-01-12 at 12 15 59" src="https://user-images.githubusercontent.com/6804901/104308277-ae5a0d00-54d0-11eb-8d89-8ba1d67048c6.png">